### DISCUSSION TITLE
Hello Everybody

### DISCUSSION THREAD

**Discussion by WebShells** [opensourcepos/opensourcepos#3687] [Team Posts]
Hello Everybody, you've been missed guys.
Share your updates, is everyone in a good health ?

**Comment by odiea** [comment]
Good here in Colorful Colorado.

**Comment by objecttothis** [comment]
@WebShells I checked here in opensourcepos/org/teams but I don't see more details about your status.  I vaguely remember seeing something but I didn't see it on gitter either.  Perhaps you can send me a link to the post so you don't have to regurgitate it.
